src/test/scala/ai/privado/exporter/JavaLanguageEgressTest.scala

```diff
@@ -0,0 +1,91 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, contact support@privado.ai
+ *
+ */
+
+package ai.privado.exporter
+
+import ai.privado.cache.S3DatabaseDetailsCache
+import ai.privado.entrypoint.PrivadoInput
+import ai.privado.exporter.HttpConnectionMetadataExporter
+import ai.privado.languageEngine.java.JavaTaggingTestBase
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import ai.privado.tagger.sink.RegularSinkTagger
+
+import scala.collection.mutable
+
+class JavaLanguageEgressTest extends JavaTaggingTestBase {
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+  }
+
+  override val javaFileContents: String =
+    """
+      |package com.twilio;
+      |
+      |import org.springframework.beans.factory.annotation.Autowired;
+      |import org.springframework.http.ResponseEntity;
+      |import org.springframework.stereotype.Service;
+      |import org.springframework.web.client.RestTemplate;
+      |
+      |@Service
+      |public class NasaApodService {
+      |
+      |    private final RestTemplate restTemplate;
+      |
+      |    @Autowired
+      |    public NasaApodService(RestTemplate restTemplate) {
+      |        this.restTemplate = restTemplate;
+      |    }
+      |
+      |    public NasaApodResponse getNasaApod(String apiKey) {
+      |        String login = "api/v1/login";
+      |        String userMetaPath = "api/v1/" + "user/meta";
+      |        String userId = "123";
+      |        String apiKey = "key";
+      |        String formatted = String.format("https://vineet%s/hellow", "api/hellow");
+      |        String apiUrl = "https://abc.com/planetary/apod?api_key=" + apiKey;
+      |        ResponseEntity<NasaApodResponse> responseEntity = restTemplate.getForEntity(apiUrl, NasaApodResponse.class);
+      |        ResponseEntity<NasaApodResponse> responseEntity = restTemplate.getForEntity(apiUrl, NasaApodResponse.class);
+      |        return responseEntity.getBody();
+      |    }
+      |}
+      |""".stripMargin
+
+  "Java code egresses" should {
+    "collect egress url for java code" in {
+      val propertyExporter          = new HttpConnectionMetadataExporter(cpg, ruleCache)
+      val egressesFromLanguageFiles = propertyExporter.getLiteralsFromLanguageFiles
+      egressesFromLanguageFiles.size shouldBe 5
+      egressesFromLanguageFiles shouldBe List(
+        "api/v1/login",
+        "api/v1/user/meta",
+        "https://vineet%s/hellow",
+        "api/hellow",
+        "https://abc.com/planetary/apod?api_key=apiKey"
+      )
+    }
+  }
```

Comment:
Can we add a test case for multiple assignments, I believe it will not give expected result. 
Ex - `"api/v1/" + "user/meta" + "/profile"`